docs/implementation.rst

```diff
@@ -0,0 +1,61 @@
+Implementation of ploneorg.addonlisting 
+===============================================================
+
+Plone Dexterity Content type is used to store the add-on data locally.
+
+The custom content type addOn consists of the following fields:
+
+* title – Name of the add-on and it is a mandatory field.
+* description - A one-line description for the add-on (up to 120 characters); if not specified,
```

Comment:
Documentation is not code, line length might not fit for doc texts. 
Please read and apply https://docs.plone.org/about/rst-styleguide.html 

Line Break at each "." and not a line break within a sentence.